sdk/keyvault/azure-keyvault-keys/src/main/java/module-info.java

```diff
@@ -2,9 +2,10 @@
 // Licensed under the MIT License.
 
 module com.azure.security.keyvault.keys {
-    requires transitive com.azure.core;
 
     requires commons.codec;
+    requires java.xml.crypto;
+    requires com.azure.core;
```

Comment:
Why did we remove the `transitive` requirement? Someone using this module will need that dependency.